.travis.yml

```diff
@@ -35,16 +35,16 @@ language: python
 matrix:
   include:
     - python: 2.6
-      env: NUMPY=numpy==1.6 MOCK=mock
+      env: NUMPY=numpy==1.6
     - python: 2.7
-      env: MOCK=mock PANDAS=pandas
+      env: PANDAS=pandas MOCK=mock
     - python: 3.3
     - python: 3.4
     - python: 3.5
     - python: 2.7
       env: TEST_ARGS=--pep8
     - python: 2.7
-      env: BUILD_DOCS=true MOCK=mock
+      env: BUILD_DOCS=true
```

Comment:
Mock is a dependency of building the docs because we mock out PyQT4 for the api docs. Perhaps we should switch the docs build to 3.5 instead?